src/com/walmartlabs/lacinia/pedestal.clj

```diff
@@ -59,18 +73,24 @@
                       (update-in [:response :body] cheshire/generate-string))
                   context)))}))
 
-(def require-graphql-content-interceptor
+(def require-content-type-interceptor
```

Comment:
I'm starting to question if this step is necessary; the alternate is that when a request is not graphql or json, the request processing will fail when the query doesn't exist in the parse interceptor.  Perhaps we could extend that step to build a failure message that discusses content type when the query is missing.